connector/connect/common/src/main/protobuf/spark/connect/ml.proto

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = 'proto3';
+
+package spark.connect;
+
+import "spark/connect/expressions.proto";
+import "spark/connect/relations.proto";
+import "spark/connect/ml_common.proto";
+
+option java_multiple_files = true;
+option java_package = "org.apache.spark.connect.proto";
+
+
+message Evaluator {
+  string name = 1;
+  Params params = 2;
+  string uid = 3;
+}
+
+
+message MlCommand {
+  oneof ml_command_type {
+    Fit fit = 1;
+    ModelAttr model_attr = 2;
+    ModelSummaryAttr model_summary_attr = 3;
+    LoadModel load_model = 4;
+    SaveModel save_model = 5;
+    Evaluate evaluate = 6;
+  }
+
+  message Fit {
+    Stage estimator = 1;
+    Relation dataset = 2;
+  }
+
+  message Evaluate {
+    Evaluator evaluator = 1;
+  }
+
+  message LoadModel {
```

Comment:
will `LodeModel` and `SaveModel` also support load/save estimator/transformer?

Reply:
Sure.

Reply:
I will rename it to `LoadStage` to support estimator too.

Reply:
For estimator, we need define a new LoadEstimator command